akka-actor-tests/src/test/scala/akka/actor/DynamicAccessSpec.scala

```diff
@@ -25,6 +25,13 @@ class TestClassWithDefaultConstructor extends TestSuperclass {
 class DynamicAccessSpec extends AnyWordSpec with Matchers with BeforeAndAfterAll {
   val system = ActorSystem()
 
+  // TODO DOTTY, yes I know it's wrong
+  implicit val pos: org.scalactic.source.Position = new org.scalactic.source.Position(fileName = "", filePathname = "", lineNumber = 1)
+
+  // TODO DOTTY
+  protected override def runTest(testName: String, args: org.scalatest.Args): org.scalatest.Status = akka.testkit.ScalatestRunTest.scalatestRunTest(testName, args)
```

Comment:
This is again regarding ScalaTest, I would love to know if there is a better way of doing it.
cc.  @bvenners @cheeseng